docs/src/content/en/docs/workflows-vnext/inngest-workflow.mdx

```diff
@@ -151,30 +96,25 @@ Register the workflow with Mastra and configure the Inngest API endpoint:
 ```ts showLineNumbers copy filename="src/mastra/index.ts"
 import { Mastra } from '@mastra/core/mastra'
 import { serve as inngestServe } from '@mastra/inngest'
+import { incrementWorkflow } from './workflows'
+import { inngest } from './inngest'
 import { PinoLogger } from '@mastra/loggers'
-import { Inngest } from 'inngest'
-import { incrementWorkflow } from './workflows/inngest-workflow'
-import { realtimeMiddleware } from '@inngest/realtime'
-import { serve } from '@hono/node-server'
-import { createHonoServer } from '@mastra/deployer/server'
-
 
-// Create an Inngest instance with realtime middleware for development
-const inngest = new Inngest({
-  id: 'mastra',
-  baseUrl: `http://localhost:8288`,
-  isDev: true,
-  middleware: [realtimeMiddleware()],
-})
 
 // Configure Mastra with the workflow and Inngest API endpoint
 export const mastra = new Mastra({
   vnext_workflows: {
     incrementWorkflow
   },
   server: {
+    // The server configuration is required to expose the Inngest integration endpoint.
+    // This server acts as a bridge between Mastra and Inngest, allowing:
+    // - Inngest to discover and register Mastra workflows
+    // - Inngest to send events to trigger workflow executions
+    // - Real-time communication for workflow monitoring and updates
     host: '0.0.0.0',
     apiRoutes: [
+      // This API route is used to register the Mastra workflow (inngest function) on the inngest server
```

Comment:
What does the inngest serve function do? This should probably be described on a higher level above where we describe how the integration works.

But also worth noting here with some detail.